controls/cis_ocp_1_4_0/section-5.yml

```diff
@@ -11,33 +11,39 @@ controls:
     controls:
     - id: 5.1.1
       title: Ensure that the cluster-admin role is only used where required
-      status: pending
-      rules: []
+      status: partial
```

Comment:
same as with the other PRs my only question is about the partial state

Reply:
Updating to manual since that's more appropriate given the operator isn't really doing anything with this rule.